src/inverseRobot.js

```diff
@@ -7,7 +7,22 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+  const keys = Object.keys(robot);
+
+  for (const elem of values) {
+    if (values.indexOf(elem) !== values.lastIndexOf(elem)) {
+      return null;
+    }
+  }
+
+  const resultObj = {};
+
+  for (let i = 0; i < values.length; i++) {
+    resultObj[values[i]] = keys[i];
```

Comment:
i think you can write this row in the first loop and delete the second one. but before that move   const resultObj = {};
up

Reply:
That makes sense, thanks!